### DISCUSSION TITLE
正文和参考文献中的生僻字问题

### DISCUSSION THREAD

**Discussion by sirius-vincent** [sjtug/SJTUThesis#772] [提问]
1、正文中的生僻字，例如旻（min2）无法显示，通过拆开偏旁部首上下组合勉强打出来了；
2、参考文献中有的作者名字含有生僻字，例如旸，应该如何打出？

有无更好更方便的方式单独处理生僻字？

**Comment by skyzh** [comment]
相关的讨论有很多了，可以自己翻一下
